### PR TITLE
snap: official image / test

### PR DESCRIPTION
**Pull request by TriplEight**
snapcraft has the official image `snapcore/snapcraft`, makes no sense to maintain our version.

### CODE REVIEW

**Comment by 5chdn** [comment]
while at it, could you replace `libdb` with `libdb5.3` in 

```
./scripts/snap/snapcraft.template.yaml
```

File: scripts/snap/snapcraft.template.yaml [hunk 1]
```diff
@@ -50,8 +50,9 @@ parts:
       cp -v ethkey $SNAPCRAFT_PART_INSTALL/usr/bin/ethkey
       cp -v ethstore $SNAPCRAFT_PART_INSTALL/usr/bin/ethstore
       cp -v whisper $SNAPCRAFT_PART_INSTALL/usr/bin/whisper
-    stage-packages: [libc6, libudev1, libstdc++6, cmake, libdb]
+    stage-packages: [libc6, libudev1, libstdc++6, cmake, libdb5.3]
   df:
+    source:         /bin/
     plugin:         nil
     stage-packages: [coreutils]
     stage:          [bin/df]
```
Comment (TriplEight) [inline scripts/snap/snapcraft.template.yaml]: @devops-parity 
```
Setting target machine to 'amd64'
Failed to pull source: unable to determine source type of './artifacts/x86_64-unknown-linux-gnu'.
Check that the URL is correct or consider specifying `source-type` for this part. See `snapcraft help sources` for more information.
ERROR: Job failed: exit code 1
```
I'm stuck here.
https://gitlab.parity.io/parity/parity-ethereum/-/jobs/115724

**Comment by 5chdn** [comment]
Are you still working on this? @TriplEight

**Comment by TriplEight** [comment]
@5chdn , @devops-parity is working on it.

**Comment by General-Beck** [comment]
need for test after merging.

**Comment by TriplEight** [comment]
```
icon:               ./scripts/snap/icon.png

parts:
  desktop-icon:
    source:         ./scripts/snap
    plugin:         nil
    override-build:        |
      mkdir -p $SNAPCRAFT_PART_INSTALL/usr/share/applications
      mkdir -p $SNAPCRAFT_PART_INSTALL/usr/share/pixmaps
      cp -v ./parity.desktop $SNAPCRAFT_PART_INSTALL/usr/share/applications/
      cp -v ./icon.png $SNAPCRAFT_PART_INSTALL/usr/share/pixmaps/
  parity:
    source:         ./artifacts/x86_64-unknown-linux-gnu
    plugin:         nil
    override-build:        |
      mkdir -p $SNAPCRAFT_PART_INSTALL/usr/bin
      cp -v parity $SNAPCRAFT_PART_INSTALL/usr/bin/parity
      cp -v parity-evm $SNAPCRAFT_PART_INSTALL/usr/bin/parity-evm
      cp -v ethkey $SNAPCRAFT_PART_INSTALL/usr/bin/ethkey
      cp -v ethstore $SNAPCRAFT_PART_INSTALL/usr/bin/ethstore
      cp -v whisper $SNAPCRAFT_PART_INSTALL/usr/bin/whisper
    stage-packages: [libc6, libudev1, libstdc++6, cmake, libdb5.3]
Setting target machine to 'amd64'
Failed to pull source: unable to determine source type of './artifacts/x86_64-unknown-linux-gnu'.
Check that the URL is correct or consider specifying `source-type` for this part. See `snapcraft help sources` for more information.
ERROR: Job failed: exit code 1
```
guess this could change after merging.

**Pull request merged**